src/service/variable-source.js

```diff
@@ -44,15 +80,40 @@ const LOAD_EVENT_END = 'loadEventEnd';
  * @return {!Promise<ResolverReturnDef>}
  */
 export function getTimingDataAsync(win, startEvent, endEvent) {
-  let readyPromise = loadPromise(win);
-  if (startEvent === LOAD_EVENT_END || endEvent === LOAD_EVENT_END) {
+  // Fallback to load event if we don't know what to wait for
+  const startWaitForEvent =
+    NAV_TIMING_WAITFOR_EVENTS[startEvent] || WAITFOR_EVENTS.LOAD;
+  const endWaitForEvent = endEvent
+    ? NAV_TIMING_WAITFOR_EVENTS[endEvent] || WAITFOR_EVENTS.LOAD
+    : null;
+
+  let waitForEvent;
+  if (!endWaitForEvent) {
+    waitForEvent = startWaitForEvent;
```

Comment:
This can just be moved to the `null` above.

Reply:
good point. done